visual-tests/tests/expand-dropdown-menu.spec.js

```diff
@@ -0,0 +1,31 @@
+import { test, expect } from '@playwright/test';
+import path from 'path';
+import { helpers } from '../imports/helpers';
+
+const stylesToAdd = [
+  helpers.cssFiles.cookieInfo,
+  helpers.cssFiles.dynamicDataFreeze
+];
+
+test(helpers.testTitle(path.basename(__filename)), async({ page }, workerInfo) => {
+  helpers.init(workerInfo);
+  await page.goto(helpers.testURL);
+  await expect(page).toHaveTitle(helpers.expectedPageTitle);
+  stylesToAdd.forEach(item => page.addStyleTag({ path: helpers.cssPath(item) }));
+
+  const table = page.locator(helpers.selectors.mainTable);
+
+  await table.waitFor();
+  helpers.mainTableFirstColumn = table.locator(helpers.selectors.mainTableFirstColumn);
+  helpers.tbody = table.locator(helpers.selectors.mainTableBody);
+  helpers.thead = table.locator(helpers.selectors.mainTableHead);
+
+  const changeTypeButton = table.locator(helpers.findDropdownMenuExpander({ col: 2 }));
```

Comment:
```suggestion
  await table.waitFor();

  const changeTypeButton = table.locator(helpers.findDropdownMenuExpander({ col: 2 }));
```
Some of the elements are not used within the test so I suggesting to remove them.

Reply:
Hi! Yes, as I mentioned here: https://github.com/handsontable/handsontable/pull/10165#discussion_r1069027662
it was done in this way intentionally - I wanted to keep some things declared and always just available to make testers life easier.

Reply:
The same thing appears in few files - it was done intentionally as a compromise to make tester's life easier, now should be modified as we discussed.